Rethinking_2/environment.yml

```diff
@@ -5,6 +5,8 @@ dependencies:
   - jupyter
   - mkl-service
   - pip
+  - causalgraphicalmodels
+  - daft
```

Comment:
I think they aren't on conda, are they?
In that case, just write them down in the pip section below, like watermark